tracer/test/Datadog.Trace.Tests/SpanContextTests.cs

```diff
@@ -16,30 +16,30 @@ public class SpanContextTests
         [Fact]
         public void OverrideTraceIdWithoutParent()
         {
-            const ulong expectedTraceId = 41;
+            var expectedTraceId = (TraceId)41;
             const ulong expectedSpanId = 42;
 
             var spanContext = new SpanContext(parent: null, traceContext: null, serviceName: "service", traceId: expectedTraceId, spanId: expectedSpanId);
 
             spanContext.SpanId.Should().Be(expectedSpanId);
-            spanContext.TraceId.Should().Be(expectedTraceId);
+            spanContext.TraceId128.Should().Be(expectedTraceId);
```

Comment:
It's me, hi, I'm the problem, it's me 👋🏼 

This is one more file where I think we may want to still assert SpanContext.TraceId, in addition to the new SpanContext.TraceId128

Reply:
Ok, I'll add it!